content/tutorials/realtime-attendance-system-iot.textile

```diff
@@ -0,0 +1,262 @@
+---
+title: IoT based realtime attendance system
+section: tutorials
+index: 66
+alt_title: IoT based realtime attendance system
+excerpt: Learn how to build an IoT based realtime attendance system for Slack using Ably and Zapier
+category: introduction
+platform: web
+authors:
+- author_name: Ably team
+  author_bio: ""
+  author_profile_url: "https://www.ably.io"
+  author_image: "tutorials/ably.png"
+level: medium
+reading_time: 30
+languages:
+  - cplusplus
+tags:
+  - ably-features
+ably_product: reactor
+---
+
+The Ably Reactor provides a way to trigger events and to stream data from Ably's pub/sub channels. In this tutorial, we'll see how to build an IoT based relatime attendance system that sends a message to a Slack channel upon detecting an access card. We'll use "Ably's MQTT adapter":/mqtt to communicate with the IoT device and Zapier to send messages to Slack.
+
+h2(#step1-prerequisites). Step 1 - Prerequisites
+
+For this tutorial, we'll be using the following hardware components:
+
+1. "Adafruit Feather HUZZAH ESP866":https://learn.adafruit.com/adafruit-feather-huzzah-esp8266/overview - WiFi enabled board that works with the Arduino software
+2. "MFRC522 RFID sensor":https://playground.arduino.cc/Learning/MFRC522/
+3. "Breadboard (Optional)":https://en.wikipedia.org/wiki/Breadboard
+4. "Jump wires":https://en.wikipedia.org/wiki/Breadboard#Jump_wires
+
+and the following software components:
+
+1. "The Arduino IDE":https://www.arduino.cc/en/main/software
+2. "An account with Ably":https://www.ably.io/signup (free or paid)
+3. "An account with Zapier":https://zapier.com/ (free or paid)
+4. "An account with Slack":https://slack.com/intl/en-gb/
+
+h2(#step2-setting-up-esp8266). Step 2 - Connecting the Adafruit HUZZAH ESP8266 board with RFID-MFRC522
+
+The Adafruit Feather Huzzah ESP8266 board comes with an in-built WiFi component that enables it to communicate with other entities over the internet. To connect this board with the RFID component, you'll need to set it up with the following pin wiring:
+
+<a href="/images/tutorials/iot-attendance/circuit-diagram.png" target="_blank">
+  <img src="/images/tutorials/iot-attendance/circuit-diagram.png" style="width: 100%" alt="Circuit diagram">
+</a>
+
+MFRC522 - Huzzah Pins
+* RST  - 4
+* SDA   - 5
+* MOSI - 13 
+* MISO - 12 
+* SCK  - 14 
+* 3V - 3.3V
+* GND - GND
+
+Caution: If you try to connect the power source of the RFID card to 5V instead of 3.3V, it might fry up.
```

Comment:
Caution: Do not connect the power source of the RFID card reader to the 5V power output (BATT or USB) as it might fry the reader. Be sure to connect it to 3.3V.